src/odemis/acq/test/calibration_test.py

```diff
@@ -352,5 +353,49 @@ def test_compensate_out(self):
                 self.assertEqual(vo * dcalib[0], vc)
 
 
+TIME_RANGE_TO_DELAY_EX = {1e-09: 7.99e-09,
+                          2e-09: 9.63e-09,
+                          5e-09: 3.32e-08,
+                          1e-08: 4.59e-08,
+                          2e-08: 6.64e-08,
+                          5e-08: 1.02e-07,
+                          1e-07: 1.69e-07,
+                          2e-07: 3.02e-07,
+                          5e-07: 7.31e-07,
+                          0.0001: 0.000161,
+                          0.0002: 0.00032,
+                          0.0005: 0.000798,
+                          0.01: 0.0154}
+
+
+class TimeRangeDelayTest(unittest.TestCase):
```

Comment:
This could use a small explanation of what this test class does. I don't know much about this code and wouldn't expect something with the name `TimeRangeDelayTest` to test writing and reading a csv.